### DISCUSSION TITLE
Pearson correlation in rowCorCpp?

### DISCUSSION THREAD

**Discussion by RegnerM2015** [GreenleafLab/ArchR#910] [Questions / Documentation]
Hi @rcorces and @jgranja24 

Is the pearson correlation used in the rowCorCPP function for addPeak2Genelinks?

Thanks!

**Comment by rcorces** [comment]
Yes - the rowCorCPP function is the Pearson correlation. I just annotated this in https://github.com/GreenleafLab/ArchR/commit/ef6fd035fe75329aca19d14f3c656503ff3a834e
